### DISCUSSION TITLE
Possibility to update Lidl Gateway with EmberZNet 7.4.3.0 ?

### DISCUSSION THREAD

**Discussion by jnilo1** [Koenkk/zigbee2mqtt#24360] [General]
A lot of us are happy users of the hacked Lidl Gateway updated with EmberZnet 6.7.8.0 version thanks to the initial work of [Paul Banks](https://paulbanks.org/projects/lidl-zigbee/).
Unfortunately unless we are able to update it with a EmberZnet 7.4 we will have to stick to the no longer maintained Z2M **ezsp** driver and we won't be in a position to use the newer **ember** driver.
I recently installed Silicon Labs Simplicity Studio V5 and discovered that two applications were proposed and associated with the [EFR32MG1B232F256GM48](https://www.silabs.com/wireless/zigbee/efr32mg1-series-1-socs/device.efr32mg1b232f256gm48?tab=specs) which is the chip behind the TYZS4 Tuya chip used by the Lidl Gateway.
One of these is a RCP-UART configuration. See the attached project configuration report.
I feel it could be a way to update our Lidl Gateway but since I am not an expert at all I leave that to feedbacks & suggestions from this community.
[Project report.pdf](https://github.com/user-attachments/files/17397216/Project.report.pdf)

**Comment by gpaesano** [comment]
Hi , interesting. This is what i have tried. I will set according to your project report ASAP and let you know.

**Comment by geduxas** [comment]
any news for this?

**Reply by gpaesano** [reply]
Have a look at this thread: https://github.com/zigpy/zigpy/discussions/650#top

**Reply by geduxas** [reply]
@gpaesano could you point me to right direction, i have now read everything and didn't get anything what i am looking for. I am now on V8 but it is not supported in ember..

**Reply by gpaesano** [reply]
So far I am not aware on an ember version fully working with the silvercrest.
The latest working zigbee firmware is the one attached.
Someone (me as well) has compiled a new version but the booting process stops aand need to be started manually via a Jump in the serial console. There is also a multi protocol version but I do not suggest it as only the radio module of the device is used.
[NCP_UHW_MG1B232_678_PA0-PA1-PB11_PA5-PA4.zip](https://github.com/user-attachments/files/18262136/NCP_UHW_MG1B232_678_PA0-PA1-PB11_PA5-PA4.zip)

**Reply by gpaesano** [reply]
Eventually we did it! Now we ha a 7.4.5 ember NCP firmware for the silvercrest working fine. have a look https://github.com/user-attachments/files/18932376/TYZS4_ncp-uart-hw_5.zip

**Reply by geduxas** [reply]
Wow, nice! Where did you get it? You build it?

**Reply by gpaesano** [reply]
yes with simplicity studio, see feedback for someone it is working very well for other people so and so

**Comment by jnilo1** [comment]
This issue is solved.
Check [this github](https://github.com/jnilo1/hacking-lidl-silvercrest-gateway/)
